cfg/windows.cfg

```diff
@@ -5098,6 +5098,70 @@ HFONT CreateFont(
       <not-uninit/>
     </arg>
   </function>
+  <!-- HLOCAL WINAPI LocalAlloc(
+  _In_ UINT   uFlags,
+  _In_ SIZE_T uBytes); -->
+  <function name="LocalAlloc">
+    <noreturn>false</noreturn>
+    <returnValue type="HLOCAL"/>
+    <arg nr="1">
+      <not-uninit/>
```

Comment:
I would suggest `<valid>0:</valid>` here.

Reply:
Ok, i see no problem with that. I guess that mostly some binary or'ed unknown defines will be used for this but it will not harm to add `<valid>0:</valid>` and maybe catch a bug.